src/formats/PDB.cpp

```diff
@@ -232,6 +232,12 @@ void PDBFormat::read_HELIX(string_view line) {
         return;
     }
 
+    // Fix 1HTQ
+    if (start > end) {
```

Comment:
how about inverting `start` and `end` in this case (and still emit the warning)? If the file contains an `HELIX` record, it is very likely to come from the PDB archive, and users might be interested in the secondary structure. 

Plus I could not find anything in the standard saying that `start` had to be lower than `end`

Reply:
The PDB standard is a bit of a mess, residue IDs can be negative! Also, it is a bit more complicated than swapping the `start` and `end` ids because the residue IDS in the PDB file are not sorted properly. For example, the `504` residue is between the `351` and `352` residues and the way it's implemented would cause more residues to be labeled than appropriate.  In fact, other software (e.g. PyMOL) doesn't interpret this residue at all.  So, I think emitting a warning and not assigning the group is the best solution for now.

Reply:
ok, sounds good